config.pri

```diff
@@ -2,7 +2,7 @@
 # default values if they're not set on build time
 # for a more detailed description, see INSTALL.TXT
 
-CONFIG += c++11
+CONFIG += c++1z
```

Comment:
This line should be moved below the Qt version checks to avoid a confusing no-such-CONFIG-value error when the Qt version is too old.

Reply:
We can use the following code from *qtcreator.pri* for better compatibility:
```qmake
# include c++1z as an alias for c++17 for compatibility for older Qt versions
CONFIG += c++17 c++1z
```